packages/react-app/src/components/global/Header/Menu.tsx

```diff
@@ -0,0 +1,105 @@
+import ThemeToggle from '../../parts/ThemeToggle';
```

Comment:
Split the Nav bar components and menu items into a separate file for readability, this is the menu file